content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -123,10 +125,18 @@ spec:
 
 ### Class
 
-A volume snapshot can request a particular class by specifying the name of a
+A *volume snapshot* can request a particular class by specifying the name of a
 [VolumeSnapshotClass](/docs/concepts/storage/volume-snapshot-classes/)
 using the attribute `snapshotClassName`.
-Only VolumeSnapshotContents of the requested class, ones with the same `snapshotClassName`
-as the VolumeSnapshot, can be bound to the VolumeSnapshot.
+Only *volume snapshot contents* of the requested class, ones with the same `snapshotClassName`
```

Comment:
`VolumeSnapshotContents`?

Reply:
There is no object called "VolumeSnapshotContents".